sdk/python/lib/test/langhost/util.py

```diff
@@ -238,10 +247,11 @@ def run_test(self,
             # Tear down the language host process we just spun up.
             langhost.process.kill()
             stdout, stderr = langhost.process.communicate()
+            monitor.server.stop(0)
```

Comment:
Out of curiosity, why did you move this earlier?

Reply:
We don't need the monitor or engine alive once the langhost has finished, just felt tidy to do all the stopping in one place.